patterns/1-initial/introducing-metrics-in-innersource.md

```diff
@@ -32,17 +34,30 @@ This pattern applies very widely from InnerSource initiatives in their infancy s
 * Some organizations in some countries may face extra complexity when introducing metrics as the countries may not allow tracking individuals.
 * There might be a learning curve in the discussion about metrics. And perhaps the tools do not support the InnerSource metrics we are looking for.
 
+In favour:
+
+* Management needs to understand how InnerSource is impacting development. Metrics allow for accurate visualizations.
+* There are several industry standards for monitoring the software development process (DORA, CHAOSS, ...).
```

Comment:
```suggestion
* There are several industry standards for monitoring the software development process (e.g. [DORA](https://cloud.google.com/blog/products/devops-sre/using-the-four-keys-to-measure-your-devops-performance), [CHAOSS](https://chaoss.community/)).
```

I don't want to assume that everybody knows these metrics, so I added to links to relevant resources.
Especially for DORA I am not sure if there is an authoritative source on this, but I figured Google SRE should be a good start.